frontend/src/scenes/llm-observability/llmObservabilityLogic.tsx

```diff
@@ -0,0 +1,93 @@
+import { actions, kea, path, reducers, selectors } from 'kea'
+
+import { NodeKind, TrendsQuery } from '~/queries/schema'
+import { PropertyMathType } from '~/types'
+
+import type { llmObservabilityLogicType } from './llmObservabilityLogicType'
+
+export const LLM_OBSERVABILITY_DATA_COLLECTION_NODE_ID = 'llm-observability-data'
+
+const INITIAL_DATE_FROM = '-24h' as string | null
```

Comment:
nit:  I feel like our default should be 30 days for the overall dashboard

Reply:
Sure, makes sense to me!